docs/zh_cn/build/android.md

```diff
@@ -0,0 +1,197 @@
+# Android 下构建方式
+
+- [Android 下构建方式](#Android 下构建方式)
+  - [源码安装](#源码安装)
+    - [安装构建和编译工具链](#安装构建和编译工具链)
+    - [安装依赖包](#安装依赖包)
+      - [安装 MMDeploy SDK 依赖](#安装-MMDeploy-SDK-依赖)
+    - [编译 MMDeploy](#编译-mmdeploy)
+      - [编译 SDK](#编译-sdk)
+        - [编译选项说明](#编译选项说明)
+        - [编译 SDK 库](#编译-sdk-库)
+        - [编译 SDK Demo](#编译-sdk-demo)
+
+---
+MMDeploy 为 android 平台提供交叉编译的构建方式.
+
+MMDeploy converter 部分在 linux 平台上执行,SDK 部分在 android 平台上执行.
+
+MMDeploy 的交叉编译分为两步:
+
+1. 在 linux 平台上构建 MMDeploy converter. 请根据 [How to build linux](./linux.md) 进行构建.
+
+2. 使用 android 工具链构建 MMDeploy SDK.
+
+本文档仅提供在 linux 平台上使用 android 工具链进行交叉编译构建 MMDeploy SDK 的方法.
+
+## 源码安装
+
+### 安装构建和编译工具链
+
+- cmake
+
+    **保证 cmake的版本 >= 3.14.0**. 如果不是,可以参考以下命令安装 3.20.0 版本. 如要获取其他版本,请参考 [这里](https://cmake.org/install)
+
+    ```bash
+    sudo apt-get install -y libssl-dev
+    wget https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0.tar.gz
+    tar -zxvf cmake-3.20.0.tar.gz
+    cd cmake-3.20.0
+    ./bootstrap
+    make
+    sudo make install
+    ```
+
+- ANDROID NDK 19+
+
+    **保证 android ndk 的版本 >= 19.0**. 如果不是,可以参考以下命令安装 r23b 版本. 如要获取其他版本,请参考 [这里](https://developer.android.com/ndk/downloads)
+
+    ```bash
+    wget https://dl.google.com/android/repository/android-ndk-r23b-linux.zip
+    unzip android-ndk-r23b-linux.zip
+    cd android-ndk-r23b-linux.zip
+    ```
+
+### 安装依赖包
+
+#### 安装 MMDeploy SDK 依赖
+
+如果您只对模型转换感兴趣,那么可以跳过本章节.
+<table>
+<thead>
+  <tr>
+    <th>名称 </th>
+    <th>安装方式 </th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>spdlog </td>
+    <td>
+<pre><code>
+git clone git@github.com:gabime/spdlog.git
+cd spdlog
+mkdir -p build
+cd build
+cmake -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_TOOLCHAIN_FILE=${PATH_TO_NDK}/build/cmake/android.toolchain.cmake -DANDROID_ABI="arm64-v8a" -DANDROID_PLATFORM=android-30 ..
+make install DESTDIR=${YOUR_PATH}
+</code></pre>
+   </td>
+  </tr>
+  <tr>
+    <td>OpenCV<br>(>=3.0) </td>
+    <td>
+<pre><code>
+wget https://github.com/opencv/opencv/releases/download/${OPENCV_VERSION}/opencv-${OPENCV_VERSION}-android-sdk.zip
+</code></pre>
+    </td>
+
+  </tr>
+  <tr>
+    <td>ncnn </td>
+    <td>ncnn 是支持 android 平台的高效神经网络推理计算框架</br>
+  <b> 目前, MMDeploy 支持 ncnn 的 20211208 版本, 且必须使用<code>git clone</code> 下载源码的方式安装</b><br>
+<pre><code>
+git clone -b 20211208 git@github.com:Tencent/ncnn.git
+cd ncnn
+mkdir -p build
+cd build
+cmake -DCMAKE_TOOLCHAIN_FILE=${PATH_TO_NDK}/build/cmake/android.toolchain.cmake -DANDROID_ABI="arm64-v8a" -DANDROID_PLATFORM=android-30 -DNCNN_VULKAN=ON -DNCNN_DISABLE_EXCEPTION=OFF -DNCNN_DISABLE_RTTI=OFF ..
+make install
+</code></pre>
+   </td>
+  </tr>
+</tbody>
+</table>
+
+### 编译 MMDeploy
+#### 编译 SDK
+
+##### 编译选项说明
```

Comment:
It can refer to `linux.md` for compile option

Reply:
I think the compile option in linux.md, especially "说明", is much different in android.md because only ncnn backend uses on android platform, so I adjusted some contents in this file.

Reply:
Please sync the build doc structure according to `build/linux.md` in PR #203